### PR TITLE
Bump to private branch to display more info in error messages

### PR DESCRIPTION
**Pull request by nikhita**
Ref: https://github.com/nikhita/go-git/pull/1

For https://github.com/kubernetes/publishing-bot/issues/162

/assign @dims @sttts

### CODE REVIEW

File: vendor/gopkg.in/src-d/go-git.v4/storage/filesystem/dotgit/dotgit.go [hunk 1]
```diff
@@ -596,7 +596,7 @@ func (d *DotGit) checkReferenceAndTruncate(f billy.File, old *plumbing.Reference
 		return err
 	}
 	if ref.Hash() != old.Hash() {
-		return fmt.Errorf("reference has changed concurrently")
+		return fmt.Errorf("reference has changed concurrently for %s from %s to %s", old.Name().String(), old.Hash().String(), ref.Hash().String())
```
Comment (sttts) [inline vendor/gopkg.in/src-d/go-git.v4/storage/filesystem/dotgit/dotgit.go]: can you debug.PrintStack() here?

File: vendor/gopkg.in/src-d/go-git.v4/storage/filesystem/dotgit/dotgit_setref.go [hunk 2]
```diff
@@ -74,7 +74,7 @@ func (d *DotGit) setRefNorwfs(fileName, content string, old *plumbing.Reference)
 		}
 
 		if ref.Hash() != old.Hash() {
-			return fmt.Errorf("reference has changed concurrently")
+			return fmt.Errorf("reference has changed concurrently for %s from %s to %s", old.Name().String(), old.Hash().String(), ref.Hash().String())
```
Comment (sttts) [inline vendor/gopkg.in/src-d/go-git.v4/storage/filesystem/dotgit/dotgit_setref.go]: and also here

File: vendor/gopkg.in/src-d/go-git.v4/storage/filesystem/dotgit/dotgit.go (hunk 1 above)
Comment (nikhita) [reply vendor/gopkg.in/src-d/go-git.v4/storage/filesystem/dotgit/dotgit.go]: Done.

**Pull request merged**